app/models/collection.rb

```diff
@@ -55,6 +57,7 @@ def purl
     end
 
     after_transition to: :version_draft, do: CollectionObserver.method(:after_update_published)
+    after_transition on: :update_metadata, do: CollectionObserver.method(:on_update_metadata)
```

Comment:
This is unnecessary, there already is a hook that you should be using called after_update_published

Reply:
fixed (wasn't sure on friday whether i should be using that same hook, or whether i should've caught all metadata updates)